out/notebooks/splunk.ts

```diff
@@ -129,7 +129,7 @@ export function dispatchSpl2Module(service: any, spl2Module: string, app: string
     namespace = '';
     app = app || 'search'; // default to search app
     // Get last statement assignment '$my_statement = ...' -> 'my_statement' 
-    const statementMatches = [...spl2Module.matchAll(/\$([a-zA-Z0-9_]+)[\s]*=/gm)];
+    const statementMatches = [...spl2Module.matchAll(/^\s*\$([a-zA-Z0-9_]+)[\s]*=/gm)];
```

Comment:
seems to be still matching inside doc comment, are we cool with that for now? 
<img width="540" alt="Screenshot 2023-07-12 at 2 26 03 PM" src="https://github.com/splunk/vscode-extension-splunk/assets/107712809/d065921d-ba2b-4327-88e0-b7d3a935558b">

Reply:
My bad. We're targetting fix for single line comments for now